src/euphorie/client/browser/login.py

```diff
@@ -126,6 +127,8 @@ def transferGuestSession(self):
         )
         for session in sessions:
             session.account_id = new_account.id
+            session.touch()
```

Comment:
Calling `.touch` does two things:

1. sets the last modifier
2. updates the section modified time

See https://github.com/euphorie/Euphorie/blob/af3e98fdae4b199cb6962e8e2c75d0dff6e14aa4/src/euphorie/client/model.py#L881-L883

While you want the first, I am not really sure the second is needed.

I wonder if we just can do:
```
session.last_modifier_id = new_account.id
```
without  changing the modification date of the session.

Reply:
None of this is “needed”. That's missing the point. I know there are at least four different ways of handling this. It's hard to decide if you don't tell me any reason to do it differently.

Reply:
Discussed on a call, we were not able to agree.
@pilz Please jump in the discussion